### PR TITLE
🌐 Add French translation for `docs/fr/docs/tutorial/query-params-str-validations.md`

### PR DESCRIPTION
**Pull request by Smlep**
Hey everyone! 👋

Here is the PR to translate the **query params and string validations**  page of the tutorial documentation into french.
See the french translation tracking issue [here](https://github.com/tiangolo/fastapi/issues/1972).

Thanks for the reviews👌

### CODE REVIEW

**Review by krocdort — APPROVED** [review]

File: docs/fr/docs/tutorial/query-params-str-validations.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+On utilise alors l'argument `deprecated=True` de `Query` :
+
+```Python hl_lines="18"
+{!../../../docs_src/query_params_str_validations/tutorial010.py!}
+```
+
+La documentation l'affichera bien comme déprécié (`deprecated`) :
+
+<img src="/img/tutorial/query-params-str-validations/image01.png">
+
```
Comment (JulianMaurin) [inline docs/fr/docs/tutorial/query-params-str-validations.md:285]: Missing section `Exclude from OpenAPI`
[see](https://fastapi.tiangolo.com/tutorial/query-params-str-validations/?h=string+validations#exclude-from-openapi)

File: docs/fr/docs/tutorial/query-params-str-validations.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+## Pour résumer
+
+Il est possible d'ajouter des validateurs et métadonnées pour vos paramètres.
+
+Validateurs génériques et métadonnées :
```
Comment (JulianMaurin) [inline docs/fr/docs/tutorial/query-params-str-validations.md:290]:
```suggestion
Validateurs et métadonnées génériques:
```

**Review by JulianMaurin — COMMENTED** [review]
Hey @Smlep ! 
Well done and thanks a lot for the work 💪 

I did many suggestions but some are more important than others, feel free to decline if you think that is not a good move. For example, I think that we have to avoid using the third person of singular (`on`) in favour of a formal `nous` ou `vous`.

The main issue is that we are missing two recent chapters.

Keep me in touch for the second round 🚀

File: docs/fr/docs/tutorial/query-params-str-validations.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+    Par exemple, `List[int]` vérifiera (et documentera) que la liste est bien entièrement composée d'entiers. Alors qu'un simple `list` ne ferait pas cette vérification.
+
+## Déclarer des métadonnées supplémentaires
+
+On peut aussi ajouter plus d'informations sur le paramètre.
+
+Ces informations seront incluses dans le schéma OpenAPI généré et utilisées par la documentation interactive ou les potentiels outils externes utilisés.
+
+!!! note
+    Gardez en tête que les outils externes utilisés ne supportent pas forcément tous parfaitement OpenAPI.
```
[the remark this answers is not in the document]
Comment (Smlep) [reply docs/fr/docs/tutorial/query-params-str-validations.md:232]: Maybe
```suggestion
    Gardez en tête que les outils externes utilisés supportent chacun OpenAPI à differents niveaux.
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit b93bee0dba9f836b824ccac9b14fcf53a9a12641 at: https://649a0f71acd72b0d3761e966--fastapi.netlify.app

**Review by axel584 — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you @Smlep! 🍰 

And thanks for the reviews @JulianMaurin, @axel584, @krocdort 💃

**Pull request merged**